doc/src/stdlib/numbers.md

```diff
@@ -102,11 +102,20 @@ The `BigFloat` type implements arbitrary-precision floating-point arithmetic usi
 
 ```@docs
 Base.precision
+Base.MPFR.precision(::Type{BigFloat})
```

Comment:
Apparently the first does not list all methods: http://docs.julialang.org/en/latest/stdlib/numbers.html#Base.precision